src/components/SitelinksSearchbox/SitelinksSearchbox.astro

```diff
@@ -0,0 +1,28 @@
+---
+import { getSearchPathname, getOpenSearchName, queryParamName } from '@lib/search';
+
+const { locale } = Astro.params;
+
+const url = new URL(getSearchPathname(locale), Astro.site);
+
+const script = `
```

Comment:
I finally figured out how I would do this:

- define the `schema` as a JS(ON) object, so that you can read and manipulate it as such, rather than as an HTML string.
- use a `script` tag in the template using the `schema` as its HTML.
- rename the whole component to `StructuredData.astro` as this schema contains more than just the `SitelinksSearchbox`. This also makes more sense when we extend it with more data in the future.
- add a `README.md` file explaining the `SearchAction` / `SitelinksSearchbox` feature with a link to relevant docs (like we do in other components)
- use the `<StructuredData />` component in the `<head>` of the `Default.astro` template.

```astro
---
const searchUrl = new URL(getSearchPathname(locale), Astro.site);
const searchAction = {
  "@type": "SearchAction",
  "target": {
    "@type": "EntryPoint",
    "urlTemplate": "${ searchUrl }?${ queryParamName }={searchTerms}"
  },
  "query-input": "required name=searchTerms"
}

const schema = {
  "@context": "https://schema.org",
  "@type": "WebSite",
   "name": "${getOpenSearchName(locale)}",
   "url": "${Astro.site}",
   "potentialAction": searchAction, // or directly inline here, just showing how we could now easily compose things
}
---
<script 
  is:inline 
  type="application/ld+json" 
  set:html={ JSON.stringify(schema, null, 2) } 
/>
```